src/test/java/step2/domain/LottoTest.java

```diff
@@ -0,0 +1,51 @@
+package step2.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class LottoTest {
+
+    @DisplayName("생성된 로또 번호는 6개이다.")
+    @Test
+    void generate() {
+        Lotto lotto = LottoFactory.generate();
+        int size = lotto.size();
+        assertThat(size).isEqualTo(6);
+    }
+
+    @DisplayName("생성된 로또 번호는 1~45중 하나다.")
+    @ParameterizedTest
+    @MethodSource("generateLottoNumbers")
+    void lottoNumberRangeCheck(int lottoNumber) {
+        assertThat(lottoNumber).isGreaterThan(0);
```

Comment:
특정 범위를 테스트한다면 `isBetween()`을 활용해보시는 방법도 있다는 점 참고차 말씀드려 봅니다.
```suggestion
        assertThat(lottoNumber).isBetween(1, 45);
```